docs/user/project_metadata.md

```diff
@@ -30,21 +30,44 @@ PyPI currently supports several ways of verifying project URLs. When a URL is ve
     PyPI package owner, and does not imply any additional safety about that
     URL or any other relationship to the project in question.
 
-### Self-links
+The following subsections specify the different types of URLs that can be verified.
 
-Any URL to the project on PyPI will be considered verified.
+#### Self-links
 
-### Via Trusted Publishing
+Any URL to the project on PyPI will be considered verified. For example, the project page for `pip` will mark all of the
```

Comment:
```suggestion
PyPI considers any project URL pointing on PyPI as verified. For example, the project page for `pip` will mark all of the
```

Reply:
I think this changes the meaning. Saying "PyPI considers any project URL pointing ~on~ to PyPI as verified` is too broad ("pointing to PyPI" is too generic)

 Maybe we could say "PyPI considers any project URL pointing to that project's PyPI page as verified" ?

Reply:
I think this is OK as is.

Reply:
> Maybe we could say "PyPI considers any project URL pointing to that project's PyPI page as verified" ?

That sounds good to me!